cookbooks/future/periodic_box_gmg.prm

```diff
@@ -0,0 +1,119 @@
+set Dimension = 2
```

Comment:
Not sure this file is necessary if we have the test. The other cookbook (periodic_box.prm) has been sitting around since 2014 and was not turned into a proper cookbook yet, so I guess this would be neither. Maybe just remove (I know we talked about it yesterday, but I wasnt aware of what you were intending to add).